models/src/main/scala/coop/rchain/models/rholang/sorter/ExprSortMatcher.scala

```diff
@@ -259,7 +259,7 @@ private[sorter] object ExprSortMatcher extends Sortable[Expr] {
       case gs: GString => ScoredTerm(e, Node(Score.STRING, Leaf(gs.value))).pure[F]
       case gu: GUri    => ScoredTerm(e, Node(Score.URI, Leaf(gu.value))).pure[F]
       case GByteArray(ba) =>
-        ScoredTerm(e, Node(Score.EBYTEARR, Leaf(ba.toStringUtf8))).pure[F]
```

Comment:
This is not backwards-compatible

Reply:
That's fine, we are including other breaking changes like channels-map removal. We only need to ensure that we can start test net with single-parent until BM part is finished.